### ISSUE TITLE
BIG BUG in BlockLayered with IE 9

### ISSUE THREAD

**Issue by Captain-FLAM** [thirtybees/thirtybees#611]
During my tests, I found that this line block the normal function of blocklayered with IE 9 :
```
function addSlider(type, data, unit, format) {
    // NEW !
    sliderList = sliderList.filter(slider => slider.type !== type);
```
With "Syntax Error" about the > sign ...

I don't have time to dig it, so in waiting I just commented it out (as it comes from TB, this line is not in PS 1.6.1.20)

**Comment by getdatakick** [comment]
Arrow function is not [supported](https://caniuse.com/#search=arrow%20function) by IE yet

**Comment by getdatakick** [comment]
Fixed by commit thirtybees/blocklayered@fa45992d46df54583b6c682c4140284a2dbb2d47
